res/i18n/fr/labels.json

```diff
@@ -371,7 +371,7 @@
 	"ratio": "Ratio",
 	"ratio_high": "Ratio haut",
 	"ratio_low": "Ratio bas",
-	"ratio_knee:db": "Ratio,\genoux (dB)",
+	"ratio_knee:db": "Ratio,\genou (dB)",
```

Comment:
Here we get propably improper new line escape sequence, please fix this with '\n'.

Reply:
done